server/schedule/checker/rule_checker.go

```diff
@@ -78,7 +78,13 @@ func (c *RuleChecker) fixRange(region *core.RegionInfo) *operator.Operator {
 	if len(keys) == 0 {
 		return nil
 	}
-	return operator.CreateSplitRegionOperator("rule-split-region", region, 0, pdpb.CheckPolicy_USEKEY, keys)
```

Comment:
@disksing Please confirm whether the rule checker needs to be updated.

Reply:
it's ok